zipline-gradle-plugin/src/main/kotlin/app/cash/zipline/gradle/ZiplinePlugin.kt

```diff
@@ -39,14 +40,39 @@ class ZiplinePlugin : KotlinCompilerPluginSupportPlugin {
     version = BuildConfig.KOTLIN_PLUGIN_VERSION,
   )
 
+  override fun apply(target: Project) {
+    super.apply(target)
+
+    // If the build has compileDevelopmentExecutableKotlinJs, create developmentExecutableZipline.
+    // Symmetrically for production.
+    target.tasks.whenTaskAdded { task ->
```

Comment:
This forces all tasks to be configured. I'm going to recommend a change of approach below, but if we were keeping this strategy the lazy API to use is `configureEach`.

Reply:
New technique is this:

```
    val extension = project.extensions.findByType(KotlinMultiplatformExtension::class.java)
      ?: return

    extension.targets.withType(KotlinJsIrTarget::class.java).all { kotlinTarget ->
      ...
    }
```

I _think_ I need `.all {}` so I can create my task early enough in the lifecycle.